### DISCUSSION TITLE
posts/2023/07/naozhoudao-tenth-day/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#16] [General]
# posts/2023/07/naozhoudao-tenth-day/

https://guozheng.rbind.io/posts/2023/07/naozhoudao-tenth-day/

**Comment by liechi** [comment]
>不要叫我相信
流星会带来好运
那颗悲伤的逃兵
怎么能够实现我许过的愿

---《谁的眼泪在飞》 by  孟庭苇

所以，三楼的小妹妹早已看穿了一切：）

**Reply by residualsun1** [reply]
哈哈，原来流星还可以看作“悲伤的逃兵”，如果小二妹真这样想，就别有意味了。
